CONTRIBUTING.md

```diff
@@ -224,9 +224,19 @@ The design review process is as follows:
   if (GVFSPlatform.Instance.KernelDriver.EnumerationExpandsDirectories)
   ```
 
-- *Add new interfaces when it makes sense for the product, not simply for unit testing*
+- *Testability of a class should be a priority in the design of a class and during code reviews.*
 
-  When a class needs to be mocked (or have a subset of its behavior mocked), prefer using virtual methods instead of adding a new interface.  VFS4G uses interfaces when multiple implementations of the interface exist in the product code.
+  When writing new code, aim to follow existing coding guidelines. However, when exiting guidelines and testability goals of a design conflict, optimize for testability.
```

Comment:
> when exiting guidelines and testability goals of a design conflict

- `exiting` -> `existing`
- Does "existing coding guidelines" here refer to other guidelines that might conflict with testability?
- Would it be fair to reword this to something like "When guidelines conflict, optimize for testability goals.  The only exception is that testability changes should not negatively impact performance critical paths."